modules/ROOT/pages/database-management.adoc

```diff
@@ -41,33 +41,33 @@ The algorithm allows the branch to retain a configurable number of revisions (re
 === Controls
 You can vary the number of retained revisions using the Configuration File's {configuration-schema-database--xref--revs-limit} setting.
 
-So, for example, with a `revs_limit` of 1,000 the algorithm will keep the last 1,000 revisions in the shortest non-tombstoned branch and remove any others from that branch.
+So, for example, with a `revs_limit` of 1,000 the algorithm keeps the last 1,000 revisions in the shortest non-tombstoned branch and remove any others from that branch.
 
 [NOTE]
---
-Do not set `revs_limit` below 100 when `allow_conflicts = true` +
-*Otherwise* ... you may adversely affect the conflict resolution process, as there may be insufficient revision history to resolve a given conflict.
---
+---
+Sync Gateway 4.x uses automatic conflict resolution with Last Write Wins strategy, allowing `revs_limit` values as low as 1.
+In earlier versions, setting `revs_limit` below 100 when `allow_conflicts = true` can lead to data loss during conflict resolution due to insufficient revision history.
+---
 
 
 [#lbl-rtcons]
 === Constraints
-The default and minimum values of `revs_limit` are dependent on whether {configuration-schema-database--xref--allow-conflicts} is  True or False.
+The default and minimum values of `revs_limit` are dependent on whether {configuration-schema-database--xref--allow-conflicts} is True or False.
 
 The presence of multiple unresolved conflicts in a revision tree can impair the pruning process.
-It may result in obsolete revisions not being pruned or in the premature pruning of revisions.
+This may prevent pruning of obsolete revisions or cause premature pruning of revisions.
```

Comment:
Since there are no longer obsolete revisions or allow_conflicts=true, we can remove this.